src/towncrier/test/test_builder.py

```diff
@@ -66,3 +66,15 @@ def test_dots_in_ticket_name_and_counter(self):
             parse_newfragment_basename("baz.1.2.feature.3", ["feature"]),
             ("2", "feature", 3),
         )
+
+    def test_strip(self):
+        self.assertEqual(
```

Comment:
Minor comment

would be nice to have a docstring explaing the scope/purpose/expectation of this test :)

This can serve as the developer documentation for this feature.

Reply:
https://github.com/twisted/towncrier/pull/278/files#diff-b7254cf4717de60e5d2e57233db4f5cbb47871439b71f2280bed581ff9fcf851R71-R73

```python
        """Leading spaces and subsequent leading zeros are stripped
        when parsing newsfragment names into ticket numbers etc.
        """
```